src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -262,10 +286,13 @@ public string ServiceName
 
                 Close();
                 _name = value;
-                _displayName = "";
+                _displayName = string.Empty;
```

Comment:
This is fine, but there is no reason to prefer one over the other. They compile to the same thing. As far as I know we haven't established a convention in this repo.

https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxNMAfAAgJgIwCwAULgAzG4DMABAN7HWPVY1Z4BszALNQIIAUASjoMmYtgE5+AImmCA3KMYBfJdTUtmHbnxxCRRMeLxSAyjAQBLAHYBzAHQBRMAAcYATwVrlY4sqA==